dev/com.ibm.ws.org.apache.wss4j.policy/bnd.overrides

```diff
@@ -11,14 +11,14 @@
 #     IBM Corporation - initial API and implementation
 #*******************************************************************************
 -include= ~../cnf/resources/bnd/rejar.props
-bVersion=1.0
+bVersion=2.3.0
 
 Bundle-Name: org.apache.wss4j.wss4j.policy
 Bundle-Description: org.apache.wss4j.wss4j.policy; version=2.3.0
-Bundle-SymbolicName: com.ibm.ws.org.apache.wss4j.policy.2.3.0
+Bundle-SymbolicName: com.ibm.ws.org.apache.wss4j.policy
 Bundle-ActivationPolicy: lazy
 
 WS-TraceGroup: WSS4J
 
 Export-Package: \
- org.apache.wss4j.policy.*
+ org.apache.wss4j.policy.*;version="2.3.0"
```

Comment:
Could we use this variable `bVersion` here so we only have to update the variable if we change the version export values?